pkg/api/types.go

```diff
@@ -215,6 +215,8 @@ type VolumeSource struct {
 	DownwardAPI *DownwardAPIVolumeSource `json:"downwardAPI,omitempty"`
 	// FC represents a Fibre Channel resource that is attached to a kubelet's host machine and then exposed to the pod.
 	FC *FCVolumeSource `json:"fc,omitempty"`
+	// AzureFile represents an Azure File Service mount on the host that shares a pod's lifetime
```

Comment:
`shares a pod's lifetime` -> I know this is the comment used on a bunch of existing plugins. But it doesn't really seem correct. The whole point of a these network attached storage volumes is that they persist beyond the lifecycle of a pod. You could argue that the mount itself exists for the duration of a pod lifetime.

In short, if you can come up with better wording for this, that would be great, if not, no biggie.